### DISCUSSION TITLE
How to pass arguments for ls/lsd

### DISCUSSION THREAD

**Discussion by FahimAnayet** [gazorby/fifc#41] [Q&A]
I use [lsd](https://github.com/lsd-rs/lsd) instead for exa/eza. I do not have exa on my path, so the plug in fall back to ls. Currently I am diong this `set -U fifc_exa_opts --depth 2 --tree --color=always` but it's not working.

**Comment by krissh-codes** [comment]
As a hacky workaround, you can create a symlink to lsd for exa like:
`ln -s {path-to-lsd} /usr/local/exa`

**Reply by FahimAnayet** [reply]
I've switched to nushell :p  but I later used exa for this.
